woocommerce-services.php

```diff
@@ -209,6 +209,33 @@ function wpcom_static_url($file) {
 			return set_url_scheme( $url );
 		}
 
+		/**
+		 * environment_check function.
+		 *
+		 * @access public
```

Comment:
`@public` in redundant and not necessary https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods